docs/software/devtools/linaro/linaro-ddt.md

```diff
@@ -27,29 +27,33 @@ mpicxx -g test_cpu.cpp test_gpu.o -o myexe
 ### Launch Linaro DDT
 
 To use the DDT client with uenv, it must be launched in `Manual Launch` mode
-(assuming that it is connected to Alps via `Remote Launch`):
+(assuming that it is connected to [Alps][ref-alps] via `Remote Launch`):
 
-=== "on local machine"
+=== "On local machine"
 
     Start DDT, and connect to the target cluster using the drop down menu for `Remote Launch`.
+    If you don't have a target cluster,
+    the [`linaro-forge` uenv documentation][ref-uenv-linaro] explains how to set up the connection the first time.
 
-    Click on `Manual launch`, set the number of processes to listen to, then wait for the slurm job to start (see the "on Alps" tab).
+    Click on `Manual launch`, set the number of processes to listen to, then wait for the Slurm job to start 
+    (see the "on Alps" tab for how to start the Slurm job).
 
     <img src="https://raw.githubusercontent.com/jgphpc/cornerstone-octree/ddt/scripts/img/ddt/0.png" width="600" />
 
-=== "on Alps"
+=== "On Alps"
 
     Log into the system and launch with the `srun` command:
 
-    ```terminal
-    # start a session with both the PE used to build your application
-    # and the linaro-forge uenv mounted
-    > uenv start prgenv-gnu/24.11:v1,linaro-forge/24.1.1:v1 --view=prgenv-gnu:default
-    > source /user-tools/activate
-
-    > srun -N1 -n4 -t15 -pdebug ./cuda_visible_devices.sh   ddt-client   ./myexe
+    ```console
+    $ uenv start prgenv-gnu/24.11:v1,linaro-forge/24.1.1:v1 --view=prgenv-gnu:default # (1)!
```

Comment:
Just checking, is `$` vs `>` convention, preference, something else? We can probably add this to the style guide if this is what we prefer.

Reply:
`>` doesn't seem to trigger the correct highlighting, while `$` does. In my experience, `console` is a bit flaky.

Reply:
That's a good catch. I did some testing and opened https://github.com/eth-cscs/cscs-docs/pull/74 with some changes and a style guide for code blocks that we can try to follow going forward.